Lib/slapdtest/_slapdtest.py

```diff
@@ -582,10 +602,17 @@ def _open_ldap_conn(self, who=None, cred=None, **kwargs):
         """
         return a LDAPObject instance after simple bind
         """
+        ldap_conn = self._make_ldap_object(**kwargs)
+        ldap_conn.simple_bind_s(who or self.server.root_dn, cred or self.server.root_pw)
+        return ldap_conn
+
+    def _make_ldap_object(self, **kwargs):
```

Comment:
If subclasses are supposed to call this, it should be documented.
Perhaps it would be better to add an overridable `bind=True` class attribute?